### PR TITLE
unit coverage

### PR DESCRIPTION
**Pull request by vglalitha**

### CODE REVIEW

File: dropwizard-demo/test/com/test/ControllerTest.java [hunk 1]
```diff
@@ -57,26 +54,8 @@ public void testcase_noTweetsFound() {
...
-        arrayList.add("hlo");
-        Response expectedTweet = Response.ok(arrayList).build();
-        Response actualTweet = tweetPost.GetTweets();
-        Assert.assertEquals(expectedTweet.getStatus(), actualTweet.getStatus());
-        Assert.assertEquals(expectedTweet.getStatus(),actualTweet.getStatus());
     }
 
-    @Test
-    public void testcase_noTweetsFound() throws TwitterException {
-        when(brsConfiguration.configurationBuilder()).thenReturn(new ConfigurationBuilder());
```
Comment (PoojaMohite-Khoros) [inline dropwizard-demo/test/com/test/ControllerTest.java:71]: Y these tests were deleted?

**Review by PoojaMohite-Khoros — CHANGES_REQUESTED** [review]

File: dropwizard-demo/test/com/test/ControllerTest.java (hunk 1 above)
Comment (vglalitha) [reply dropwizard-demo/test/com/test/ControllerTest.java:71]: This tests are added mutliple times, so deleted Pooja.

Comment (PoojaMohite-Khoros) [reply dropwizard-demo/test/com/test/ControllerTest.java:71]: cool

**Review by PoojaMohite-Khoros — APPROVED** [review]
image
Report submitted

**Pull request merged**